content/articles/integrate-facebook-ads-into-your-android-application-in-android-studio/index.md

```diff
@@ -72,10 +72,10 @@ Once done, you are ready for the integration process in the Android studio.
 
 ### Integration in Android Studio
 
-Don't have an Application? No need to worry since you can set-up an simple Application in a few steps below.
+Don't have an Application? No need to worry since you can set up a simple application in a few steps below.
 
 #### Step 1: Create a New Project
-If you don't have an application created already, open your Android Studio and create a new project with Ads activity. Since it is a test Application, you can use it without altering. This [article](https://www.section.io/engineering-education/first-android-App/) can be very useful in helping you with proper project creation.
+If you don't have an application created already, open your Android Studio and create a new project with Ads activity. Since it is a test Application, you can use it without altering it. This [article](https://www.section.io/engineering-education/first-android-App/) can be very useful in helping you with proper project creation.
```

Comment:
There's no need to include this "If you don't have an application created already" here again, you have already mentioned it above.